### ISSUE TITLE
Link 'x' contains a language reference: undefined

### ISSUE THREAD

**Issue by johnottone** [microsoft/linkcheckermd#62]
This is driving me nuts, and I cannot find a way to ignore this warning. I initially thought it might be markdownlint, because I thought markdownlint was the only extension I had installed, that was parsing my markdown for errors. I had configured for Markdown files, but, @DavidAnson suggested I look at the LinkCheckerMD, and sure enough, the warning is coming from LinkCheckerMD. 

When writing a markdown file, if I include a link or multiple links, as demonstrated below, I get a warning for each link, and it does not have an error/warning code and I can't find any information, ANYWHERE, on how to ignore it. It is kind of annoying having yellow squiggly lines everywhere on the document. Note, that if I take off the view=powershell-7.3 the links do not trigger a warning, and I could just do a find replace on all of them, but that's not really a feasible solution. All of the links below are links to Microsoft PowerShell documentation, but this same warning is triggered on any link in markdown, containing similar "URL arguments/parameters". What am I missing? Is this a bug? A feature? How can I ignore this warning either in the user settings.json, or the workspace settings.json? If I cannot ignore this warning, I will have to uninstall the LinkCheckerMD extension. Please help!!!

```markdown
- Overview
  - [What is PowerShell?](https://learn.microsoft.com/en-us/powershell/scripting/overview?view=powershell-7.3)
  - [What is a PowerShell command?](https://learn.microsoft.com/en-us/powershell/scripting/powershell-commands?view=powershell-7.3)
  - [Discover PowerShell](https://learn.microsoft.com/en-us/powershell/scripting/discover-powershell?view=powershell-7.3)
```
 
![vscode-markdown-error-20221127-01](https://user-images.githubusercontent.com/112649342/204206240-aac1d01f-9ba9-4de8-b80c-19ac7cbbb337.png)
